.travis.yml

```diff
@@ -10,10 +10,9 @@ before_install:
 - git checkout -
 script:
 - COVERAGE=y npm test
-# Run changelog-lint but only on newer versions of Node (because of syntax errors)
-- if [[ ${TRAVIS_NODE_VERSION:0:1} -ge "4" ]]; then
-    npm run changelog-lint;
-  fi
+# Run changelog-lint
```

Comment:
Since this comment doesn't add anything that the command itself doesn't already convey, you can remove it